zio-http/shared/src/main/scala/zio/http/Header.scala

```diff
@@ -1160,8 +1160,21 @@ object Header {
 
     final case class Basic(username: String, password: Secret) extends Authorization
 
-    object Basic {
+    object Basic extends HeaderType {
       def apply(username: String, password: String): Basic = new Basic(username, Secret(password))
+
+      override def name: String = "basicAuth"
```

Comment:
This is just wrong. There is no header named "basicAuth" this is "authorization".
Below as well